sips/sip-028/sip-028-sbtc_peg.md

```diff
@@ -0,0 +1,203 @@
+# SIP-028: sBTC Signer Criteria
+
+**SIP Number:** 028  
+**Title:** Signer Criteria for sBTC, A Decentralized and Programmable Asset Backed 1:1 with BTC  
+**Consideration:** Technical, Economics  
+**Type:** Standard  
+**Status:** Draft  
+**Authors:**  
+- Mårten Blankfors ([marten@trustmachines.co](mailto:marten@trustmachines.co))  
+- Daniel Jordon ([daniel@trustmachines.co](mailto:daniel@trustmachines.co))
+- Friedger Müffke ([friedger@ryder.id](mailto:friedger@ryder.id))  
+- Jesus Najera ([jesus@stratalabs.xyz](mailto:jesus@stratalabs.xyz))  
+- Jude Nelson ([jude@stacks.org](mailto:jude@stacks.org))  
+- Tycho Onnasch ([tycho@zestprotocol.com](mailto:tycho@zestprotocol.com))  
+- Andre Serrano ([andre@bitcoinl2labs.com](mailto:andre@bitcoinl2labs.com))  
+- Ashton Stephens ([ashton@trustmachines.co](mailto:ashton@trustmachines.co))  
+- Joey Yandle ([joey@trustmachines.co](mailto:joey@trustmachines.co))  
+
```

Comment:
Preamble is missing required fields:

Missing Created field. Please add the ISO 8601 date when this SIP was created.

Missing License field. Please specify the content license for this SIP.

Missing Sign-off field. Please add this field as placeholder, even if it is not yet filled in entirely.

Missing Layer field in Preamble:
Since this is a Standard-type SIP, the Layer field is required. Please specify the logical layer of the Stacks blockchain affected (e.g., 'Applications' or 'Peer Services' since this is about p2p signer network).

Missing Discussions-To field in Preamble:
Please provide a mailing list or discussion forum where ongoing discussion of the SIP takes place (e.g. sBTC Working Group discussions?)

Reply:
@aldur @andrerserrano

Reply:
```
Missing Discussions-To field in Preamble:
Please provide a mailing list or discussion forum where ongoing discussion of the SIP takes place (e.g. sBTC Working Group discussions?)
```

is this standard? i'm not seeing a precedence for this requirement. all i can find is this, which suggests it may be required *sometimes*: https://github.com/stacksgov/sips/blob/2ffa3ad9de843faf233174388e4a6ab15bc8dcab/sips/sip-000/sip-000-stacks-improvement-proposal-process.md?plain=1#L128

Reply:
not a must I think, however sip-000 itself is precedence. 
no harm in including relevant discussions which lead to this SIP standard

Reply:
ok thanks - @andrerserrano let's either add that, or explain why it's not needed for this SIP please.

Reply:
Added a link to this for the sBTC Working Group Discussions